### PR TITLE
[EV-3557] Policy Recommendation Scope resource definition

### PR DESCRIPTION
**Pull request by dimitri-nicolo**
Defines the policy recommendation scope resource

Product Version(s):

Calico Enterprise
Calico Cloud

Issue:

[EV-3557](https://tigera.atlassian.net/browse/EV-3557)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

[EV-3557]: https://tigera.atlassian.net/browse/EV-3557?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by dimitri-nicolo** [comment]
@bmckercher123 Could you please add yourself as a reviewer?

File: calico-enterprise/reference/resources/policyrecommendations.mdx [hunk 1]
```diff
@@ -18,52 +18,17 @@ import Icmp from '@site/calico-enterprise_versioned_docs/version-3.16/_includes/
...
-Policy recommendations are a kind of [staged network policy](/reference/resources/stagednetworkpolicy.mdx) that allow flows to and from a given namespace. New policy rules are generated by flows that are not captured by existing policies. 
-
-The policy name is suffixed with `-recommendation` to differentiate it from other staged network policies. The naming scheme follows the pattern: 
-
-`namespace-segmentation.<namespace>-recommendation`
-
-where `namespace-segmentation` is the predefined tier name used for recommendations. 
-
-If the Policy Recommendations resource is deleted, all recommendations with the state of learning or ignored, are also deleted. Activated policies (policies moved to the Policy Board in Manager UI) are not affected when the resource is deleted, and are handled like any other policy.
+Policy recommendation scope represents a collection of configuration options used to control the [policy recommendation](../../network-policy/recommendations/policy-recommendations.mdx) engine.
```
Comment (bmckercher123) [inline calico-enterprise/reference/resources/policyrecommendations.mdx]:
```suggestion
The policy recommendation scope is a collection of configuration options to control the [policy recommendation](../../network-policy/recommendations/policy-recommendations.mdx) engine.
```

File: calico-enterprise/reference/resources/policyrecommendations.mdx [hunk 2]
```diff
@@ -72,96 +37,19 @@ spec:
...
-## Supported operations
-
-| Datastore type           | Create/Delete | Update | Get/List | Notes |
-| ------------------------ | ------------- | ------ | -------- | ----- |
-| Kubernetes API datastore | Yes           | Yes    | Yes      |
\ No newline at end of file
+| Field                  | Description                                                  | Accepted Values | Schema | Default        |
+| ---------------------- | ------------------------------------------------------------ | --------------- | ------ | -------------- |
+| recStatus              | Defines the policy recommendation engine status.             | Enabled/Disabled |        | Disabled       |
+| selector               | Selects the namespaces for which to generate recommendations for. |                 |        | `!(projectcalico.org/name starts with ''tigera-'') && !(projectcalico.org/name starts with ''calico-'') && !(projectcalico.org/name starts with ''kube-'')`    |
```
Comment (bmckercher123) [inline calico-enterprise/reference/resources/policyrecommendations.mdx]:
```suggestion
| selector               | Selects the namespaces for generating recommendations. |                 |        | `!(projectcalico.org/name starts with ''tigera-'') && !(projectcalico.org/name starts with ''calico-'') && !(projectcalico.org/name starts with ''kube-'')`    |
```

**Review by bmckercher123 — APPROVED** [review]
Minor nits. Go ahead and copy to CC 3.17 folder.

**Pull request merged**